patches/protonvpn-gui/webkit-version.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/protonvpn_gui/view/dialog.py b/protonvpn_gui/view/dialog.py
+index e26f815..f8adf3a 100644
+--- a/protonvpn_gui/view/dialog.py
++++ b/protonvpn_gui/view/dialog.py
+@@ -3,7 +3,7 @@ import os
+ import gi
+ 
+ gi.require_version("Gtk", "3.0")
+-gi.require_version("WebKit2", "4.0")
++gi.require_version("WebKit2", "4.1")
```

Comment:
If possible this should be upstreamed with the explanation of why it is good (uses libsoup 3). Note that one can load 4.1 or 4.0 dynamically depending on what the system has, see https://github.com/lutris/lutris/pull/4554